examples/scripts/denoise_tv_iso_admm.py

```diff
@@ -9,10 +9,11 @@
 ================================
 
 This example compares denoising via isotropic and anisotropic total
-variation (TV) regularization. It solves the denoising problem
+variation (TV) regularization :cite:`rudin-1992-nonlinear`
+:cite:`goldstein-2009-split`. It solves the denoising problem
 
   $$\mathrm{argmin}_{\mathbf{x}} \; (1/2) \| \mathbf{y} - \mathbf{x}
-  \|^2 + \lambda R(\mathbf{x}) \;,$$
+  \|_2^2 + \lambda R(\mathbf{x}) \;,$$
 
 where $R$ is either the isotropic or anisotropic TV regularizer.
```

Comment:
perhaps use lowercase for the regularizer since it is not an operator?

Reply:
That would make sense, but this usage is scattered across multiple examples, so it may be better to fix it as part of a separate PR.